pipeline-modules/common/gcp/pipeline-config.yaml

```diff
@@ -39,6 +39,13 @@ inputs:
       description: Type of the VM used to run the pipeline.
```

Comment:
Can also add here in description the reference link  machine type : https://cloud.google.com/build/docs/api/reference/rest/v1/projects.builds#machinetype